java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -0,0 +1,159 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.ExpireOptions;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Generic Commands interface to handle generic commands for all server requests.
+ *
+ * @see <a href="https://redis.io/commands/?group=generic">Generic Commands</a>
+ */
+public interface GenericBaseCommands {
+
+    /**
+     * Sets a timeout on <code>key</code> in seconds. After the timeout has expired, the <code>key
+     * </code> will automatically be deleted. If <code>key</code> already has an existing <code>expire
+     * </code> set, the time to live is updated to the new value. If <code>seconds</code> is a
+     * non-positive number, the <code>key</code> will be deleted rather than expired. The timeout will
+     * only be cleared by commands that delete or overwrite the contents of <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/expire/">redis.io</a> for details.
+     * @param key The key to set timeout on it.
```

Comment:
Somewhere you emphasize the `key`, somewhere not. Do you want to fix?
```suggestion
     * @param key The <code>key</code> to set timeout on it.
```